plugins/org.python.pydev.core/tests/org/python/pydev/core/ModulesKeyTest.java

```diff
@@ -70,7 +70,7 @@ public void testToIo4() throws Exception {
     }
 
     public void testToIo3Folder() throws Exception {
-        ModulesKeyForFolder key = new ModulesKeyForFolder("bar.a", new File("f.py"));
+        ModulesKeyForFolder key = new ModulesKeyForFolder("bar.a.__init__", new File("bar/a"));
```

Comment:
Same here. I just updated the test based on the ones above and I inserted `.__init__` in the key name. I also corrected it's file path